myia/api.py

```diff
@@ -3,10 +3,10 @@
 from types import FunctionType
 from typing import Any, Callable, Dict, List
 
-from myia import parser
-from myia.anf_ir import Graph, Constant, ANFNode
-from myia.vm import VM as VM_
-from myia.prim import implementations, ops as P
+from . import parser
```

Comment:
Did we decide something regarding absolute vs. relative imports?
If so, that would be the opportunity to standardize.

Reply:
No. I'm voting for relative with this PR, and if anyone disagrees, we can debate it.

Reply:
I would stay with absolute for top-level imports (like here), but use relative within a sub-package.

The reason is that it reduces the changes required when we change files to packages or move things around.

Reply:
@abergeron Do you mean using `from myia.ir import ...` from sub-packages instead of `from ..ir import ...`? Because I've done the latter.